packages/manager/modules/web-paas/src/details/service/service.controller.js

```diff
@@ -6,6 +6,18 @@ export default class WebPassDetailsServiceCtrl {
   constructor(Alerter) {
     this.Alerter = Alerter;
     this.ADDON_TYPE = ADDON_TYPE;
+    this.WEB_PAAS_TRACK_OPEN_PSH =
+      'web-paas::project-dashboard::goto-platform-sh-project';
+    this.WEB_PAAS_TRACK_CHANGE_OFFER =
+      'web-paas::project-dashboard::change-range';
+    this.WEB_PAAS_TRACK_CHANGE_VCPU =
+      'web-paas::project-dashboard::setup-vcpus';
+    this.WEB_PAAS_TRACK_ADDON_STORAGE =
+      'web-paas::project-dashboard::upgrade-storage';
+    this.WEB_PAAS_TRACK_ADDON_LICENSE =
+      'web-paas::project-dashboard::setup-user';
+    this.WEB_PAAS_TRACK_ADDON_ENVIRONMENT =
+      'web-paas::project-dashboard::setup-additional-environments';
```

Comment:
Here, "web-paas::project-dashboard::" is a common prefix which can be extracted to route the way its done in kubernetes.